roles/keystone/tasks/federation.yml

```diff
@@ -10,8 +10,10 @@
   retries: 5
 
 - name: download mod_auth_openidc
-  get_url: url={{ keystone.federation.sp.oidc.download.url }}
-           dest=/tmp/{{ keystone.federation.sp.oidc.module_name }}
+  get_url:
+    url: "{{ keystone.federation.sp.oidc.download.url }}"
+    dest: /tmp/{{ keystone.federation.sp.oidc.module_name }}
+    checksum: "{{ keystone.federation.sp.oidc.download.checksum }}"
```

Comment:
yeah, we need to have a discussion with site controller team and see if we can easily integrate this into one of the repositories we host on package cloud. In the mean time I'll merge this.